app/src/main/java/com/aileyzhang/musicapp/activities/MainActivity.java

```diff
@@ -29,8 +30,8 @@
         ActivityCompat.OnRequestPermissionsResultCallback {
 
     private static final int SDCARD_PERMISSION_REQUEST_CODE = 1;
-
-    private ViewPager mViewPager;
+    public MainActivityContentAdapter mMainActivityContentAdapter;
+    public ViewPager mMainViewPager;
```

Comment:
sorry to nitpick again lmao, can these be private? Best to restrict access to how much is being used, or access level intended by design 👌